src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpointManagerBase.cs

```diff
@@ -85,6 +88,9 @@ protected async virtual Task ReloadServiceEndpointsAsync(IEnumerable<ServiceEndp
 
                 UpdateEndpoints(endpoints, out var addedEndpoints, out var removedEndpoints);
 
+                _accessKeyManager.AddHubServiceEndpoints(addedEndpoints);
+                _accessKeyManager.RemoveHubServiceEndpoints(removedEndpoints);
```

Comment:
I think it's better to remove endpoint after https://github.com/Y-Sindo/azure-signalr/blob/1265d783358f5ed77483a27b45918bef28d476ac/src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpointManagerBase.cs#L108, otherwise a endpoint may be removed from `AccessKeyManager` before it's removed from hub.